timeseries/src/autogluon/timeseries/models/gluonts/mx/models.py

```diff
@@ -19,40 +22,50 @@
     from gluonts.mx.model.tft import TemporalFusionTransformerEstimator
     from gluonts.mx.model.seq2seq import MQCNNEstimator, MQRNNEstimator
 
-from .abstract_gluonts import AbstractGluonTSModel
+from .callback import GluonTSEarlyStoppingCallback, TimeLimitCallback
 
 logger = logging.getLogger(__name__)
+gts_logger = logging.getLogger(gluonts.__name__)
+
+
+class AbstractGluonTSMXNetModel(AbstractGluonTSModel):
```

Comment:
Should we also add some warning / check ensuring that the user has MXNet installed?

Reply:
will do in a follow up PR